src/map/atcommand.c

```diff
@@ -7695,17 +7696,18 @@ ACMD(duel) {
 		return false;
 	}
 
-	if (!duel->checktime(sd)) {
+	diff = duel->checktime(sd);
+	if (diff > 0) {
 		char output[CHAT_SIZE_MAX];
-		// "Duel: You can take part in duel only one time per %d minutes."
-		sprintf(output, msg_fd(fd,356), battle_config.duel_time_interval);
+		// "Duel: You can take part in duel again after %d secconds."
+		sprintf(output, msg_fd(fd,356), diff);
 		clif->message(fd, output);
 		return false;
 	}
 
 	if (message[0]) {
 		if (sscanf(message, "%12u", &maxpl) >= 1) {
-			if (maxpl < 2 || maxpl > 65535) {
+			if (maxpl < 2 || maxpl > MAX_DUEL) {
```

Comment:
I thought this `MAX_DUEL` are created to set the limit of max player to duel ...
tho I never heard anyone would any that high amount of duel in server before, they would just pick a pvp map rather than create duel and invite so many peoples.

I will revert this changes.